SpeechService/Translations.cs

```diff
@@ -679,7 +679,7 @@ public static string Humanize(decimal? value)
                     case 2:
                         return "well over " + number + " " + order;
                     case 3:
-                        return "on the way to " + number + " and a half " + order;
+                        return "about " + number + " and a half " + order;
```

Comment:
Let's take for example the number 2301, so `{Humanise(2301)}`. 
nextDigit would be the 2nd integer in the complete number sequence, so it'd be equal to '3' in this example. 

The old script result would be 'On the way to two and a half thousand'.
The proposed new script result would be 'About two and a half thousand'. Since 2301 is not, in my opinion, about two and a half thousand, then I regard the proposal as less accurate than the original. 

Possible alternative: 'approaching' (e.g. 'approaching two and a half thousand')

Reply:
Your original suggestion of 'almost' would also be a better fit than 'about'.